yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnSparkHadoopUtil.scala

```diff
@@ -85,7 +85,7 @@ class YarnSparkHadoopUtil extends SparkHadoopUtil {
 
 }
 
-object YarnSparkHadoopUtil {
+private[spark] object YarnSparkHadoopUtil {
```

Comment:
I don't think we can change this to private at this point. Already public in 1.0